.github/workflows/pr-integration.yml

```diff
@@ -75,11 +75,13 @@ jobs:
         echo ::set-output name=vault-token::$VAULT_TOKEN
         echo ::add-mask::$VAULT_TOKEN
 
-    - name: Render config
-      uses: ./.github/actions/render-config
+    - name: Write config
+      id: config
+      uses: ./.github/actions/write-config
       with:
         test-env: dev
```

Comment:
Looks like this line can be removed...since `./.github/actions/write-config` no longer uses `test-env` as input.

Reply:
Thanks for catching this!